src/measure_theory/integral/set_integral.lean

```diff
@@ -393,18 +449,60 @@ end
 
 end nonneg
 
-lemma set_integral_mono_set {α : Type*} [measurable_space α] {μ : measure α}
-  {s t : set α} {f : α → ℝ} (hfi : integrable f μ) (hf : 0 ≤ᵐ[μ] f) (hst : s ≤ᵐ[μ] t) :
-  ∫ x in s, f x ∂μ ≤ ∫ x in t, f x ∂μ :=
+section tendsto_mono
+
+variables {μ : measure α}
+  [measurable_space E] [normed_group E] [borel_space E] [complete_space E] [normed_space ℝ E]
+  [second_countable_topology E] {s : ℕ → set α} {f : α → E}
+
+lemma tendsto_set_integral_of_monotone (hsm : ∀ i, measurable_set (s i))
+  (h_mono : monotone s) (hfi : integrable_on f (⋃ n, s n) μ) :
+  tendsto (λ i, ∫ a in s i, f a ∂μ) at_top (𝓝 (∫ a in (⋃ n, s n), f a ∂μ)) :=
+let bound : α → ℝ := indicator (⋃ n, s n) (λ a, ∥f a∥) in
```

Comment:
can you move the let inside the proof? I think having a pure tactic proof is more readable instead of having just one separate line like this. Same thing in the next proof.